src/View/Helper/FormHelper.php

```diff
@@ -1336,6 +1353,24 @@ protected function _optionsOptions(string $fieldName, array $options): array
         return $options;
     }
 
+    /**
+     * Get map of enum value => label for select/radio options.
+     *
+     * @param class-string<\BackedEnum> $enumClass Enum class name.
+     * @return array<int|string, string>
+     */
+    protected function enumOptions(string $enumClass): array
+    {
+        assert(is_subclass_of($enumClass, BackedEnum::class));
+
+        $values = [];
+        foreach ($enumClass::cases() as $case) {
+            $values[$case->value] = method_exists($case, 'label') ? $case->label() : $case->name;
```

Comment:
Is the idea here that enums can implement `label()` to get custom formatting?

Reply:
Yes, I have added a test case using the `ArticleStatusLabel` enum for the same.